packages/perseus-editor/src/article-editor.tsx

```diff
@@ -15,7 +15,12 @@ import SectionControlButton from "./components/section-control-button";
 import Editor from "./editor";
```

Comment:
All changes to this file will be reverted now that we're performing the conversion in Webapp.